tools/format.py

```diff
@@ -29,7 +29,6 @@
 rustfmt_extra_args = []
 if 'RUSTFMT_FLAGS' in os.environ:
```

Comment:
FYI I don't `RUSTFMT_FLAGS` any more. 
You can remove or I will do it myself.
(^-- this sounds unintentionally threatening)